src/main/kotlin/org/jitsi/nlj/util/Bandwidth.kt

```diff
@@ -137,3 +137,14 @@ val Long.mbps: Bandwidth
 fun DataSize.per(duration: Duration): Bandwidth {
     return Bandwidth(this.bits / duration.seconds.toDouble())
 }
+
+/**
+ * Returns the sum of all elements in the collection.
+ */
+fun Iterable<Bandwidth>.sum(): Bandwidth {
```

Comment:
nit: this could be `fun Iterable<Bandwidth>.sum(): Bandwidth = reduce(Bandwidth::plus)`